test/specs/elements/Input/Input-test.js

```diff
@@ -62,4 +63,23 @@ describe('Input', () => {
       })
     })
   })
+
+  describe('onChange', () => {
+    it('handles onChange', () => {
```

Comment:
This should be covered by the [conformance test for events](https://github.com/TechnologyAdvice/stardust/blob/master/test/specs/commonTests.js#L259).  It runs every component through every event handler and asserts that it was called, and was called with the appropriate event shape as the first arg.